app/controllers/users.js

```diff
@@ -109,23 +155,34 @@ var create = function create(req, res, next) {
           if (signinError) {
             next(signinError);
           } else {
-            credentials.uuid = cloudCredentials.uuid;
             credentials.token = cloudCredentials.token;
-            signupKNoTCloud(credentials, function onSignup(signupErr) {
-              if (signupErr) {
-                next(signupErr);
+            setCloudConfig(credentials.token, cloudSettings, function onSetCloudConfig(setCloudConfigErr) { // eslint-disable-line max-len
+              if (setCloudConfigErr) {
+                next(setCloudConfigErr);
               } else {
-                res.end();
+                signupKNoTCloud(credentials, function onSignup(signupErr) {
```

Comment:
Shouldn't we first call the `signupKNoTCloud` operation and then configure the connector?

Reply:
Done!